include/miopen/config.h.in

```diff
@@ -78,11 +78,65 @@
 
 // clang-format off
 #cmakedefine MIOPEN_DEFAULT_FIND_MODE @MIOPEN_DEFAULT_FIND_MODE@
+
 // clang-format on
+#cmakedefine01 MIOPEN_OFFLINE_COMPILER_PATHS_V2
+#if MIOPEN_OFFLINE_COMPILER_PATHS_V2
+// Include rocm-core header for get ROCm Install Path Method
+#include <rocm-core/rocm_getpath.h>
+
+// Include header for PATH_MAX
+#if defined(__linux__)
+#include <linux/limits.h>
+#endif
+#if defined(_WIN32)
+#include <windows.h>
+#ifndef PATH_MAX
+#define PATH_MAX MAX_PATH
```

Comment:
Don't use `MAX_PATH` on Windows to allocate for maximum path characters. This is only true for older applications and the FAT filesystem, which is not widely used today. NTFS is capable of 32K charter paths.

Reply:
These lines of code removed in latest version